hubconf.py

```diff
@@ -0,0 +1,128 @@
+"""
+Configuration file for loading pretrained models using PyTorch hub
+Usage example: torch.hub.load("castorini/howl", "hey_fire_fox")
+"""
+dependencies = ['howl', 'torch']
+
+import dataclasses
+import os
+import pathlib
+import shutil
+import typing
+import zipfile
+
+import torch
+
+import howl.context as context
+import howl.data.transform as transform
+import howl.model as howl_model
+import howl.model.inference as inference
+import howl.settings as settings
+
+from howl.settings import SETTINGS as _SETTINGS
+
+_MODEL_URL = "https://github.com/castorini/howl-models/archive/v1.0.0.zip"
+_MODEL_CACHE_FOLDER = "howl-models"
+
+@dataclasses.dataclass
+class _ModelSettings:
+    path: str # Relative to the howl-models root folder
+    model_name: str
+    vocab: typing.List[str]
+    inference_sequence: typing.List[int]
+    token_type: str
+    use_frame: bool
+
+
+def hey_fire_fox(pretrained=True, **kwargs) \
+        -> typing.Tuple[inference.InferenceEngine, context.InferenceContext]:
+    """
+    Pretrained model for Firefox Voice
+    """
+    # Audio inference settings
+    conf = _ModelSettings(path="howl/hey-fire-fox",
+                          model_name="res8",
+                          vocab=["hey", "fire", "fox"],
+                          inference_sequence=[0, 1, 2],
+                          token_type="word",
+                          use_frame=True)
```

Comment:
I think we need to design this one little more carefully.

I think what we eventually want is other people developing training their own model, uploading it to hub, and then they should update this code in a minimal way to expose their models through the pip package.

Having said that, I feel like we need a helper function that does model loading and hey_fire_fox should only be creating the setting and triggering that helper function.

So when someone else add new model, all they need to do is simply creating a new setting and triggering that helper